### PR TITLE
suite.py: allow specifying ceph sha1

### PR DESCRIPTION
**Pull request by dmick**
Signed-off-by: Dan Mick <dan.mick@redhat.com>

### CODE REVIEW

**Comment by yuriw** [comment]
👍

**Comment by zmc** [comment]
I like this - but the ``--help`` text should be updated as well.

I also wonder if we should have a unit test for the new behavior.

**Comment by athanatos** [comment]
lgtm

**Comment by dmick** [comment]
I'll investigate help and unit test

**Comment by dmick** [comment]
I think I'm done.

File: teuthology/suite.py [hunk 1]
```diff
@@ -252,12 +253,23 @@ def create_initial_config(suite, suite_branch, ceph_branch, teuthology_branch,
     else:
         kernel_dict = dict()
 
-    # Get the ceph hash
-    ceph_hash = git_ls_remote('ceph', ceph_branch)
+    # Get the ceph hash: if --cephsha1/-S is supplied, use it if
```
Comment (zmc) [inline teuthology/suite.py]: Looks like it's ``--sha1``, not ``--cephsha1``

**Pull request merged**